### PR TITLE
Clean up NumericField

### PR DESCRIPTION
**Pull request by assertchris**

### CODE REVIEW

**Comment by assertchris** [comment]
@tractorcow ping

File: forms/NumericField.php [hunk 1]
```diff
@@ -4,138 +4,196 @@
...
+	 * @param array $data
+	 *
+	 * @return $this
+	 *
+	 * @throws Zend_Locale_Exception
+	 */
 	public function setValue($value, $data = array()) {
 		require_once "Zend/Locale/Format.php";
 
-		// If passing in a non-string number, or a value
```
Comment (tractorcow) [inline forms/NumericField.php]: Not sure why this comment was removed; This was here to help explain the logic in this method for future developers and I think it still makes sense to keep that justification in place.

Comment (tractorcow) [inline forms/NumericField.php]: In 4.0 I want to also include setDataValue and loadFrom (to complement dataValue and saveInto), at which point we can move this logic out of here. ;)

**Comment by assertchris** [comment]
Fixed

**Pull request merged**